packages/DataGrid/src/DataGrid.js

```diff
@@ -10,11 +10,16 @@ import * as sc from "./DataGrid.styles";
 import Basement, { End } from "./components/Basement";
 import InfiniteScroll from "./components/InfiniteScroll";
 import { Row, HeaderRow, StickyRow, StickyHeaderRow, createItemData } from "./DataGrid.Rows";
+import getScrollbarWidth from "./helpers/getScrollbarWidth";
 
 const propTypes = {
   /** If the data cell should automatically get focus  */
   autofocus: PropTypes.bool,
+  /** Define the look for borders in the table */
+  borderType: PropTypes.oneOf(["grid", "empty", "horizontal", "vertical"]),
```

Comment:
"none" instead of "empty"?

Reply:
These options should be in a constant that is exported so I can use it something like this:
`<DataTable borderType={DataTable.BorderTypes.GRID} />`

Reply:
yes I like the idea of the constant will do.